extensions/wikia/AdEngine/js/template/porvata.js

```diff
@@ -162,6 +166,29 @@ define('ext.wikia.adEngine.template.porvata', [
 			loadVeles(params);
 		}
 
+		if (!isVideoAutoplaySupported()) {
+			log(['hop', params.adProduct, params.slotName, params], log.levels.info, logGroup);
+			slotRegistry.get(params.slotName).hop({
+				adType: params.adType,
+				source: 'porvata'
+			});
+
+			return;
+		} else {
+			// TODO remove `if` and always track success after ADEN-5109 full roll out
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)